spec/puppetserver/ca/action/prune_spec.rb

```diff
@@ -147,6 +153,41 @@
       expect(number_of_removed_duplicates).to eq(15)
     end
 
+    it 'reduces a CRL with duplicate revoked certs when called without any arguments' do
+      allow(connection).to receive(:get).and_return(offline)
+      Dir.mktmpdir do |tmpdir|
+        with_files_in tmpdir do |bundle, key, chain, conf|
+          puppet_conf = File.join(tmpdir, 'puppet.conf')
+          File.open puppet_conf, 'w' do |f|
```

Comment:
If you like, I think you can use `with_ca_in` and it passes you a `config` that you then pass into the call to `subject.run` that has all the right paths set up.

Reply:
hmm...that didn't work for me. With that default config, the crl is not getting set correctly. So I am going to leave it as it is.